src/components/MyFarm.tsx

```diff
@@ -68,16 +105,40 @@ export default function MyFarm({
           <Box flex={1}>
             <Typography>{t("lpStaked")}</Typography>
             <Typography variant="subtitle1">
-              {formattedLpStakedBalance}
+              {veSDLFeatureReady
+                ? formattedLiquidityGaugeLpStakedBalance
+                : formattedLpStakedBalance}
             </Typography>
           </Box>
           <Box flex={1}>
             <Button
               variant="outlined"
               size="large"
               fullWidth
-              disabled={amountStaked.isZero()}
-              onClick={() => unstake(amountStaked)}
+              disabled={
+                veSDLFeatureReady
+                  ? userGaugeBalance.isZero()
+                  : amountStaked.isZero()
+              }
+              onClick={
```

Comment:
Could you move this function into a `useCallback` or just move it out of the component to make it tidier? Ditto for the `onClick` above